src/main/java/mods/eln/Eln.java

```diff
@@ -4679,7 +4833,7 @@ private void registerBrush(int id) {
 
             addShapelessRecipe(emptyStack.copy(),
                 new ItemStack(Blocks.wool, 1, idx),
-                new ItemStack(Items.iron_ingot));
+                findItemStack("Iron Cable"));
```

Comment:
..fair enough, but I feel like it'll flop around in my hand. Maybe change the recipe to produce multiple outputs instead; 1 wool is actually quite a lot of wool, given 3 makes a bed.